source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Migrate to Drupal 9 on Pantheon
+subtitle: Migrate a Drupal 9 Site to Pantheon
+description: Migrate a Composer-managed Drupal 9 Site from another platform to Pantheon.
+categories: [develop]
+cms: drupal-9
+tags: [code, launch, migrate, site, updates]
+reviewed: "2021-08-20"
+layout: guide
+showtoc: true
+permalink: docs/guides/drupal-9-migration/drupal-9-to-pantheon
+anchorid: drupal-9-migration/drupal-9-to-pantheon
+editpath: drupal-9-migration/05-drupal-9-to-pantheon.md
+---
+
+In this doc, you'll migrate an existing Composer-managed Drupal 9 site from another platform to a new Drupal 9 site with Integrated Composer on Pantheon.
+
+## Will This Guide Work for Your Site?
+
+- Drupal 9
+- Composer-managed
+- Able to get a local copy of the existing site / access to a Git repository of it?
+
+## Prepare the Local Environment
+
+<Partial file="drupal-9/prepare-local-environment.md" />
+
+This doc uses the following aliases:
+
+- **Alias:** `SITE`
+  - **Site Name:** `anita-drupal`
+
+## Create a New Drupal 9 Site
+
+- Log in to your Pantheon account. If you don't have one, [create one first](https://pantheon.io/register?docs) and familiarize yourself with the [User Dashboard](/guides/quickstart/user-dashboard) before you create a new site.
+
+- Set up [SSH Keys](/ssh-keys) on your local computer and Pantheon account.
+
+## Create a Local Copy of the Old Site's Code
+
+Follow the steps in [Manually Migrate Sites to Pantheon](/migrate-manual#import-your-code) to:
```

Comment:
As I commented below, the steps are different for a Drupal 9 site. It might be better to change the "Manually Migrate Sites to Pantheon" to tell folks to just come here to this doc if they are doing a Composer-managed Drupal 9 site, and then put the specific instructions in line here.

Reply:
Ah, and in fact the "Manually Migrate Sites to Pantheon" page does already say that the instructions on that page do not work for Composer-managed sites.